src/main/java/uk/gov/justice/hmpps/prison/api/model/AgencyPrisonerPayProfile.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.hmpps.prison.api.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.math.BigDecimal
+import java.time.LocalDate
+
+@Schema(description = "Agency prisoner pay profile")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class AgencyPrisonerPayProfile(
+  @Schema(description = "Agency identifier", example = "MDI")
+  val agencyId: String?,
```

Comment:
this looks weird - why is `agencyId` here nullable when the domain model class can't be?